Tests/SearchResultsTest.swift

```diff
@@ -581,4 +581,37 @@ class SearchResultsTest: XCTestCase {
             XCTFail("Failed to construct results")
         }
     }
+    
+    func testHierarchicalFacets() {
+        json["disjunctiveFacets"] = [
+            "number.lvl0": [
+                "one": 11,
+                "two": 5,
+                "three": 5
+            ],
+            "number.lvl1": [
+                "one > ten": 4,
```

Comment:
The test in not correct. **one** has 11 values. **one > ten**  has 4 and **one > ten > hundred** has 6. This is not possible. If you see this as a tree, the sum of the leafs should equal the root. So basically for **one**, **hundred** + **eleven** should == **one**, which it's not the case here.

Reply:
{"one": 11} = {"one > ten": 4} + {"one > eleven": 1} + {"one > ten > hundred": 6}